src/components/card/TemplateCard.vue

```diff
@@ -16,7 +16,7 @@
       class="q-px-none q-py-sm text-caption ellipsis-2-lines"
       data-cy="title-container"
     >
-      {{ template.type }}
+      {{ template.name || template.type }}
```

Comment:
template name is a required string, no need `|| X`

Reply:
When HAS_BACKEND is false the data have no `name` field